updatecli/updatecli.d/inbound-agent.yml

```
@@ -0,0 +1,59 @@
+{{ $suffix := "-jdk21-nanoserver-1809" }}
+---
+name: Bump `jenkins/inbound-agent` Docker image version
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  jenkins/inbound-agent:
+    name: '[jenkins/inbound-agent] Get latest GitHub release'
+    kind: githubrelease
+    spec:
+      owner: "jenkinsci"
+      repository: "docker-agent"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      versionfilter:
+        kind: regex
+        pattern: ^\d+\.v[a-f0-9_]+-\d+$
+
+conditions:
+  testDockerImagePublished:
+    name: "Has the Docker image jenkins/inbound-agent (with the last discovered tag) been published?"
+    kind: dockerimage
+    sourceid: jenkins/inbound-agent
+    spec:
+      image: "jenkins/inbound-agent"
+    transformers:
+      - addsuffix: {{ $suffix }}
+
+targets:
+  updateInboundAgentVersion:
+    name: "Update jenkins/inbound agent version in package-windows.yaml"
+    kind: yaml
```

```suggestion
    kind: yaml
    transformers:
      - addprefix: 'jenkins/inbound-agent:'
```

Or if you want to keep the double quotes (tested with success):

```suggestion
    kind: yaml
    transformers:
      - addprefix: 'jenkins/inbound-agent:'
      - quote: true
```